server/app/models/va/Tripler.js

```diff
@@ -51,5 +51,6 @@ module.exports = {
     relationship: 'CLAIMS',
     direction: 'in',
     eager: true
-  }
+  },
+  potential_ambassador_id: 'string'
```

Comment:
Should we instead use a relationship @brodavi ?

Reply:
I can't think of any reason to make it a relationship, but sure we can if there's a use case.

Reply:
In general, its a "relationship" and not data. So in neo4j terms, we should be using "relationship". I am concerned about introducing the concept of "pseudo foreign key in RDBMS" when we store the relationship as data

Reply:
I'm not sure what you mean by "in general it's a relationship". But in Neo4j, you want to create a relationship when you want to frequently traverse nodes. Do we frequently want to traverse the ambassador to the matched tripler? If so, then relationship. If not, then it's just a useful flag.